src/lib/seam/connect/route-types.ts

```diff
@@ -1343,7 +1343,7 @@ export interface Routes {
     jsonResponse: {
       ok: boolean
       msg: 'I\u2019m one with the Force. The Force is with me.'
-      last_service_evaluation_at: string
+      last_service_evaluation_at?: string | undefined
```

Comment:
@codetheweb Why is this nullable now?

Reply:
for https://github.com/seamapi/seam-connect/pull/3648, I fixed the types enough for it to pass without changing the actual values (i.e. was was previously optional but typed incorrectly)